### PR TITLE
IfcAudioVisualAppliance Rail Import BIMQ 2021-07-21T06:30:02.9929322Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcAudioVisualAppliance

### CODE REVIEW

**Review by jmirtsch — COMMENTED** [review]
Looks ok.

File: IFC4x3/Properties/n/NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH" Name="NumberOfInterfaces" UniqueId="3b23435a-b13b-4f9c-98ef-9f837247a851" PropertyType="p_tablevalue" PrimaryDataType="IfcLabel" />
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/n/NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH/DocProperty.xml]: Re-use of NumberOfInterfaces from IfcMobileTelecommunicationsAppliance. Commit : 7ecf4d556ad6da14673e4165d92113d7757b72b9

File: IFC4x3/Properties/s/StorageCapacity_1l1lxKbmn2bAy1hWV7EsM1/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Indicate the total data storage capacity of the device. It is defined by bytes.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/StorageCapacity_1l1lxKbmn2bAy1hWV7EsM1/Documentation.md]:
```suggestion
Indicates the total data storage capacity of the device. It is defined by bytes.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_AudioVisualApplianceTypeRecordingEquipment/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Properties of a recording equipment.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_AudioVisualApplianceTypeRecordingEquipment/Documentation.md]:
```suggestion
Properties of recording equipment.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_AudioVisualApplianceTypeRecordingEquipment/DocPropertySet.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_AudioVisualApplianceTypeRecordingEquipment" UniqueId="f2db31f5-9166-85b2-b08e-bd7ab4942b0b" ApplicableType="IfcAudioVisualAppliance/USERDEFINED">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_AudioVisualApplianceTypeRecordingEquipment/DocPropertySet.xml]: We need to look at USERDEFINED for applicability.

**Review by SergejMuhic — COMMENTED** [review]
PEnum documentation.

**Review by AlexBrad1eyCT — APPROVED** [review]
ALL Good

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**